docs/src/content/en/reference/workflows/map.mdx

```diff
@@ -1,112 +1,27 @@
 ---
-title: "Reference: Workflow.map() | Building Workflows | Mastra Docs"
-description: Documentation for the `.map()` method in workflows, which maps output data from a previous step to the input of a subsequent step.
+title: "Reference: Workflow.map() | Workflows | Mastra Docs"
+description: Documentation for the `Workflow.map()` method in workflows, which maps output data from a previous step to the input of a subsequent step.
 ---
 
 # Workflow.map()
 
 The `.map()` method maps output data from a previous step to the input of a subsequent step, allowing you to transform data between steps.
 
-## Usage
+## Usage example
 
-```typescript
```

Comment:
Same comment as for `docs/src/content/en/reference/workflows/create-run.mdx` file